.github/workflows/clean_packages.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+name: Clean packages in registry
+
+on:
+  workflow_dispatch:
+  # Runs always Sunday Midnight
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  clean_all:
+    name: Cleaning older packages
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout default branch
+        uses: actions/checkout@v4
+      - name: Clean Packages
+        uses: ./.github/actions/clean_packages
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          packages: |-
+            ort
+            ort-minimal
+            ort-extended
```

Comment:
We stopped publishing ort-extended so couldn't we remove this line once we run the delete script once?

Reply:
done